inappmessaging/USERGUIDE.md

```diff
@@ -279,6 +281,25 @@ InAppMessaging.instance().closeMessage(true)
 
 **<font color="red">Note:</font> Calling this API will not increment the campaign's impression (i.e not counted as displayed).**
 
+### <a name="custom-font"></a> #3 Custom fonts for campaigns
+
+The SDK will optionally use custom fonts on campaign texts and button texts if set. The default Android system font is used if custom fonts are not added.
+
+To use custom fonts:
+1. Add the font files, `ttf` or `otf` format, to the `font` resource folder of your app.
+2. To use custom font for campaign header and body texts, set the filename to `iam_custom_font_text`.
+3. To use custom font for button texts, set the filename to `iam_custom_font_button`.
+
+Note: If you want to use custom fonts for both campaign and button texts, app should set to font files.
```

Comment:
>app should set to font files.

Is this a typo?

Reply:
Ah yes.
Thank you for noticing.
Updated this.